roles/ood/tasks/main.yaml

```diff
@@ -49,7 +49,7 @@
     path: /etc/ood/config/ood_portal.yml
     line: '{{ item }}'
   with_items:
-    - "host_regex: '{{ compute_node_glob }}.localdomain'"
```

Comment:
I like this  change, but let's see if we can also fix the edit issue with this task.  The current implementation doesn't find the lines to replace and just appends duplicated lines at the end of the config, which can lead to config errors with two different entries in the same file.